packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -88,6 +88,12 @@ class RunCommand extends RunCommandBase {
         defaultsTo: false,
         negatable: false,
         help: 'Start in a paused mode and wait for a debugger to connect.');
+    argParser.addFlag('enable-software-rendering',
```

Comment:
Are these options shared with `flutter drive` command? If not, could you add it there too?